install.sh

```diff
@@ -2,19 +2,4 @@
 
 set -e
 
-# Install bamtools
-cd bamtools
-mkdir -p build
-cd build
-cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$PWD/../ ..
-make
-make install
-cd ../
-if [ -d "lib64" ]
-then
-	mv lib64 lib
-fi
-
-# Install LRez
-cd ../
-make
+make -j"$( nproc )" PREFIX="$( pwd )"
```

Comment:
If BamTools' build is invoked via the Makefile, this `install.sh` isn't really needed anymore ;)